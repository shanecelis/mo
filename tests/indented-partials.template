With spacing
    {{> indented-partials.partial}}

    {{> indented-partials.partial}}

Without spacing
    {{> indented-partials.partial}}
    {{> indented-partials.partial}}

With text
    {{> indented-partials.partial}}
    text
    {{> indented-partials.partial}}

In a conditional
{{#thisIsTrue}}
    {{> indented-partials.partial}}
{{/thisIsTrue}}
