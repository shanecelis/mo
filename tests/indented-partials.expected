With spacing
    first line
    second line

    first line
    second line

Without spacing
    first line
    second line
    first line
    second line

With text
    first line
    second line
    text
    first line
    second line

In a conditional
    first line
    second line
